src/main/java/Presenter/playlistPresenter.java

```diff
@@ -1,4 +1,30 @@
 package Presenter;
+import OutputBoundary.newPlaylistOutputBoundary;
+import OutputData.newPlaylistOutputData;
+import ViewModel.playlistViewModel;
 
-public class playlistPresenter {
+public class playlistPresenter implements newPlaylistOutputBoundary {
+
+    String outputMessage;
+
+    final playlistViewModel viewModel;
+
+    public playlistPresenter(playlistViewModel viewModel, newPlaylistOutputData outputData){
+        this.outputMessage = outputData.getCreatedMessage();
+        this.viewModel = viewModel;
+    }
+
+    public void setOutputMessage(String newMessage){
+        this.outputMessage = newMessage;
+    }
+    //TODO implement later & ask for help on. Also add editPlaylist stuff when I have better handle on output stuff
+
+    /**
+     *
+     * @param newPlaylistOutputData message to confirm new creation of playlist
+     * @return String confirming the creation of playlist generated by the createPlaylist Use case
+     */
+    public String playlistCreationConfirmation(newPlaylistOutputData newPlaylistOutputData) {
+        return viewModel.callNewPlaylistUseCase();
```

Comment:
Why are you passing newPlaylistOutputData into the function and then not using it? Presumably viewModel.callNewPlaylistUseCase() should take newPlaylistOutputData as an input, then.